app/assets/stylesheets/item-list.scss

```diff
@@ -186,7 +186,7 @@
         text-align: center;
         padding: 100px 0px;
 
-        span.highlight {
+        span.btnhighlight {
```

Comment:
Renamed just in case, to avoid potential conflicts with code blocks.

Reply:
Maybe rename to `btn--highlight`? Also, I don't think the `span` is necessary for the selector.